apps/fio/tests/capacity/test.yml

```diff
@@ -0,0 +1,59 @@
+# TODO 
+# Change pod status checks to container status checks (containerStatuses)
+# O/P result
+
+- hosts: localhost
+  connection: local  
+
+  vars_files:
+    - test_vars.yml
+ 
+  tasks:
+   - block:
+       - name: Replace the pvc placeholder with test param in pvc spec
+         replace:
+           path: "{{ pvc_yaml_alias }}"
+           regexp: "testclaim"
+           replace: "{{ test_name }}"
```

Comment:
it seems - we are replacing claim with test_name, which doesn't seem to be correct.

Reply:
ok.. here, you are trying to use test_name itself as claim name.. should be fine